x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_requirements_page/es_requirements_page.tsx

```diff
@@ -50,20 +51,21 @@ export const RequirementItem: React.FunctionComponent<{
   );
 };
 
-const borderColor = '#d3dae6';
-
-const StyledPageBody = styled(EuiPageBody)`
-  border: 1px solid ${borderColor};
-  border-radius: 5px;
-`;
-
 export const MissingESRequirementsPage: React.FunctionComponent<{
   missingRequirements: GetFleetStatusResponse['missing_requirements'];
 }> = ({ missingRequirements }) => {
   const { docLinks } = useStartServices();
+  const theme = useEuiTheme();
+
   return (
     <WithoutHeaderLayout>
-      <StyledPageBody restrictWidth={820}>
+      <EuiPageBody
+        restrictWidth={820}
+        css={css`
+          border: 1px solid ${theme.euiTheme.colors.lightShade};
```

Comment:
We can simplify here: `border: ${euiTheme.border.thin}`

Reply:
Thank you for the quick review! Tested the suggestion and works great - pushed 👍